Cargo.toml

```diff
@@ -15,9 +15,12 @@ debug-assertions = false
 overflow-checks = false
 panic = "abort"
 
+[features]
+wasm = ["dprint-core/wasm"]
```

Comment:
This makes it so it doesn't generate the wasm exports for any rust libraries that import this crate and don't want to have public wasm exports added.

Reply:
That is not necessary since the crate is only useful as a dprint wasm plugin.

Reply:
Oh yeah, I didn't notice that. I updated the code to work from someone importing it from Rust as well. Feel free to submit edits to this PR to remove anything or do whatever.

Reply:
I think I'll just move the parser / ast bits to another crate. I'll try to do that this week, may not have the time since I'm working on something named `dprint-plugin-html` 😉